s32217-OskarNiedzwiedzki/zadanie3.txt

```diff
@@ -0,0 +1,86 @@
+<<<<<<< HEAD
```

Comment:
Po zrobieniu merge-a wypadało by jednak jeszcze konflikty rozwiązać do końca by wynikowy "kod" był prawidłowy.
Gdybyś pracował z kodem źródłowym np. w c++ to pozostawienie takiej linii skutkowało by tym, że kod nie przechodził by walidacji.